### PR TITLE
Sync upstream rn60

### PR DESCRIPTION
**Pull request by auzwang**
Sync upstream updates

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {
   "name": "react-native-signature-capture",
```
Comment (terencehuynh) [inline package.json:2]: Make sure you rename this to `@localz/...`

```suggestion
  "name": "@localz/react-native-signature-capture",
```

**Review by terencehuynh — APPROVED** [review]
I'm happy with this if you want to merge this in @auzwang

**Comment by terencehuynh** [comment]
Is this still ready to go? @auzwang

**Comment by auzwang** [comment]
@terencehuynh I haven't tested this against our apps; ended up using the old version.

**Pull request merged**